db/init.sh

```diff
@@ -25,6 +25,15 @@ EOSQL
 
 psql -U sidewalk -d sidewalk -a -f /opt/schema.sql
 
+psql -v ON_ERROR_STOP=1 -U sidewalk -d sidewalk <<-EOSQL
```

Comment:
nice! how would you feel about creating a file called `fix-auto-inc.sql` and moving these commands there and referencing it like `psql -U sidewalk -d sidewalk -a -f /opt/fix-auto-inc.sql`? this will keep the `init` script clean with setting a convention that all the "fixes" or "edits" to the schema would be in their own files. no strong preference though :)

Reply:
done!